src/core/Akka/Actor/FSM.cs

```diff
@@ -208,7 +208,7 @@ public void Cancel()
         /// </summary>
         /// <typeparam name="TS">The name of the state</typeparam>
         /// <typeparam name="TD">The data of the state</typeparam>
-        public class LogEntry<TS, TD>
+        /*TODO: this class is not used*/public class LogEntry<TS, TD>
```

Comment:
Seems to be unused, but has a nice comment header that says it can be used.
cc @Aaronontheweb